src/sparql.rs

```diff
@@ -83,7 +83,8 @@ impl<'a> QueryableDataset<'a> for &'a Hdt {
 }
 
 pub fn query<'a>(q: &str, hdt: &'a Hdt) -> Result<spareval::QueryResults<'a>, QueryEvaluationError> {
```

Comment:
@KonradHoeffner I feel a little silly not thinking of this from the get go, but do we even want an exported `sparql()` function in the `hdt` crate if it's almost a one-to-one mapping of oxigraph's exported `SparqlParser` and `QueryEvaluator`?  i.e. is the `sparql` function adding the equivalence of convenience wrapper around a function performing x+y? Purely a UX question - do we provide the function or just provide documentation on how to plug the Hdt QueryableDataset implementation into the relevant oxigraph libraries?